scripts/add_attribute.py

```diff
@@ -55,7 +55,7 @@
     ),
     "datetime": (
         "datetime.datetime",
-        "(str, unicode)",
+        None,
```

Comment:
This is a useful change, but I may pull it out into a different PR

Reply:
Would you like me to remove the commit with this change from this PR and make a new one, or do you want to take care of it?